UploadServlet.java

```diff
@@ -55,6 +60,43 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
 
 	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
 		
+		String author = request.getParameter("author");
+		String subject = request.getParameter("subject");
+		String title = request.getParameter("title");
+		String highlights = request.getParameter("highlights");
+		String abstracts = request.getParameter("abstracts");
+		
+		
+		DB db = new DB();
+		boolean checkstatus = false;
+		
+		try
+		{
+			checkstatus = db.checktitle(title);	
+		}
+		
+		catch(SQLException e)
+		{
+			e.printStackTrace();
+		}
+		
+		if(checkstatus)
+		{
+			JOptionPane.showMessageDialog(null, "Article Title is already exist please try anotherone ", "Info", JOptionPane.INFORMATION_MESSAGE);
+			
+			request.setAttribute("author", author);
+			request.setAttribute("subject", subject);
+			request.setAttribute("title", title);
+			request.setAttribute("highlights", highlights);
+			request.setAttribute("abstracts", abstracts);
+			request.getRequestDispatcher("PostArticle.jsp").forward(request, response);
+		}
+		
+		else
+		{	
+
+// ---------------------------------------------------------------------------------------------------------------
+		
 		response.setContentType("text/plai;charset=UTF=8");

```

Comment:
@hema-001 

`text/plai` ?   Did you mean `text/plain` ?

**Hui**

Reply:
Well, that is a good spot right there. I guess it slept from @mjhmohamed  last semester, and me now.

Although it is not producing any bugs for now, in the future it may does.

Thanks, I'll fix it.